Makefile

```diff
@@ -91,13 +91,14 @@ cosign: $(SRCS)
 cosign-pivkey-pkcs11key: $(SRCS)
 	CGO_ENABLED=1 go build -trimpath -tags=pivkey,pkcs11key -ldflags "$(LDFLAGS)" -o cosign ./cmd/cosign
 
+## Build cosigned binary
 .PHONY: cosigned
-cosigned: ## Build cosigned binary
+cosigned: policy-webhook 
```

Comment:
i think it complains becasue of the extra space in the end of the line

Reply:
🤔 , i thought I removed it. Thanks